packages/core/src/codewhisperer/commands/startTransformByQ.ts

```diff
@@ -226,8 +226,12 @@ export async function pollTransformationStatusUntilPlanReady(jobId: string) {
         await pollTransformationJob(jobId, CodeWhispererConstants.validStatesForPlanGenerated)
     } catch (error) {
         getLogger().error(`CodeTransformation: ${CodeWhispererConstants.failedToCompleteJobNotification}`, error)
-        transformByQState.setJobFailureErrorNotification(CodeWhispererConstants.failedToCompleteJobNotification)
-        transformByQState.setJobFailureErrorChatMessage(CodeWhispererConstants.failedToCompleteJobChatMessage)
+        if (!transformByQState.getJobFailureErrorNotification()) {
```

Comment:
To prevent the error notification / chat message from being overridden if they were already set by our pollTransformationJob function -- polling function may set these if our backend returns a specific error reason, in which case we want to show that to user, not these generic messages.